packages/app/src/files-manager/components/files-manager/files-manager.component.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import { FunctionComponent } from "Renderer/types/function-component.interface"
+import { FilesManagerContainer } from "App/files-manager/components/files-manager/files-manager.styled"
+import FilesSummary from "App/files-manager/components/files-summary/files-summary.component"
+import { Type } from "Renderer/components/core/icon/icon.config"
+import { FilesManagerTestIds } from "App/files-manager/components/files-manager/files-manager-test-ids.enum"
+
+export interface DiskSpaceCategory {
+  filesType: string
+  occupiedMemory?: number
+  filesAmount?: number
+  color: string
+  icon: Type
+}
+
+export interface MemorySpaceCategory {
+  free: number
+  full: number
+}
```

Comment:
🟢
I suggest moving the above interfaces to `files-manager.interface.tsx`

Reply:
`files-manager.interface.tsx` added [42dd729](https://github.com/mudita/mudita-center/pull/598/commits/42dd7294f4da56efac333877d32fd5442ff68173)